tailwind.config.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
       width: {
         78: "19.375rem",
       },
+      maxWidth: {
+        "6.5xl": "76.3125rem"
```

Comment:
` maxWidth: {
        90: "90rem",
      },`

This is the standard thin and you can try to add it to and test the effect on your wrapper